tree/tree/inc/TLeafD32.h

```diff
@@ -26,11 +26,11 @@
 class TLeafD32 : public TLeaf {
 
 protected:
-   Double32_t fMinimum;           ///<  Minimum value if leaf range is specified
-   Double32_t fMaximum;           ///<  Maximum value if leaf range is specified
-   Double32_t *fValue;            ///<! Pointer to data buffer
-   Double32_t **fPointer;         ///<! Address of pointer to data buffer
-   TStreamerElement *tseDouble32; ///<! StreamerElement used for TBuffer read / write
+   Double32_t fMinimum;              ///<  Minimum value if leaf range is specified
+   Double32_t fMaximum;              ///<  Maximum value if leaf range is specified
+   Double32_t *fValue;               ///<! Pointer to data buffer
+   Double32_t **fPointer;            ///<! Address of pointer to data buffer
+   TStreamerElement *fStreamingInfo; ///<! StreamerElement used for TBuffer read / write
```

Comment:
Please use fElement (a 'StreamerInfo' is a container of SteamerElement).   Thanks.